presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -112,19 +113,23 @@
     private final Map<SqlFunctionId, SqlInvokedFunction> sessionFunctions;
 
     private final Optional<Tracer> tracer;
+    private final Optional<SessionPropertyManager> sessionPropertyManager;
```

Comment:
swap the positions between `tracer` and `sessionPropertyManager` because `sessionPropertyManager` is assigned earlier than `tracer` in the constructor.